Dining/views.py

```diff
@@ -51,154 +84,84 @@ def process_date(context, day, month, year):
     return current_date
 
 
+# Todo: deprecate (possibly replace with a method on DiningListManager
 def get_list(current_date, identifier):
     """
-    Returns the dining list for the given data and identifier
-    :param current_date:
-    :param identifier:
-    :return:
+    Returns the dining list for the given data and identifier.
     """
-    # Get the dining list by the id of the association, shorthand form of the association or the person claimed
-    try:
-        if identifier is int:
-            return DiningList.objects.get(date=current_date, association_id=identifier)
-        else:
-            raise ObjectDoesNotExist("identifier is not an integer")
-    except ObjectDoesNotExist:
-        try:
-            return DiningList.objects.get(date=current_date, association__associationdetails__shorthand=identifier)
-        except ObjectDoesNotExist:
-            try:
-                return DiningList.objects.get(date=current_date, claimed_by__username=identifier)
-            except ObjectDoesNotExist:
-                # No proper identifier supplied
-                return None
+    return DiningList.objects.get(date=current_date, association__associationdetails__shorthand=identifier)
 
 
-class IndexView(View):
+class DayView(AbstractDayView):
```

Comment:
Tevens zie ik dat de slot claim closure time niet meer geimplementeerd is. Bewuste keuze of gewoon vervallen?

Reply:
Die heb ik er ergens even als todo in gezet, was er even niet over uit voor een goeie oplossing.